wsgi/odyssy/odyssy/urls.py

```diff
@@ -18,5 +18,6 @@
 
 urlpatterns = [
     url(r'^admin/?', include(admin.site.urls)),
-    url(r'^', include('basic.urls'))
+    url(r'^events/', include('events.urls')),
```

Comment:
Regex mentioned in the code is right. In fact, if we use $ in the end, it will cause problems. Let me explain. If this `^` is at the beginning of regex, it will match to all urls, which start with `events/.` If this `$` is at the end of regex, it will match to all urls, which end with `events/.` If combine both, i.e. `^`  at the beginning and  `$` at the end, it will match to only that urls that begin at `/events/` _and_ end at `events/`. Now, we have a requirement to give details of single event and url will be like `events/4`. If we use regex suggested by you, it will raise 404 error, because this url doesn't matches with any regex mentioned in `urls.py` file. So, we use a regex `^events/` which matches to any url which starts at `events/`.